kong/api/routes/kong.lua

```diff
@@ -210,6 +210,11 @@ return {
       local db_entity_name = self.params.db_entity_name
       -- What happens when db_entity_name is a field name in the schema?
       self.params.db_entity_name = nil
+      -- The validation of the certificate schema failed because snis was not part of the schema.
+      if db_entity_name == "certificates" then
+        self.params.snis = nil
```

Comment:
Just a reminder, if we remove this field, the validate result may not be reliable.

Say, we provide a `snis = "err, wrong"`, but we still return true.

Reply:
I see. We cannot guarantee the validation of fields outside the Schema. Normally, fields within the Schema will have entity_checks to ensure the validity of their values.
Please tell me if you have a good way to handle it. Thank you.

Reply:
How about inserting a warning message?

Reply:
Here?

Reply:
When snis is provided, we warn the user that this part is not validated.

Reply:
Sounds good.

Reply:
I don't particularly like this approach either. The original idea was to add a transient attribute to the schema to mark fields that can be validated but whose values won't be inserted into the database.

Reply:
Not a blocker for the PR.